src/fw/system/rtc_registers.h

```diff
@@ -45,6 +45,9 @@
 // zero them out and give up.
 #define NRF_RETAINED_REGISTER_CRC               31
 
+extern void retained_write(uint8_t id, uint32_t value);
+extern uint32_t retained_read(uint8_t id);
```

Comment:
add `bootbits.h`?

Reply:
This "feels like" it's a function of RTC registers rather than of bootbits (and that `retained` got absorbed into bootbits is an implementation detail, rather than an API feature)?

Reply:
chatted in person, OK to waive